<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>631</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 161, 255);
font: 16pt &quot;Tahoma&quot;;
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border-radius: 20px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>431</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Welcome to the Resume Maker App</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>you can select the language</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="EngButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 117, 140);</string>
    </property>
    <property name="text">
     <string>English</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PrButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 117, 140);</string>
    </property>
    <property name="text">
     <string>Persian</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
